website/docs/dbt-cli/installation.md

```diff
@@ -129,5 +129,5 @@ Then, install dbt from GitHub source
 ```shell
 git clone https://github.com/fishtown-analytics/dbt.git
 cd dbt
-pip install -r editable_requirements.txt
+pip install -r requirements.txt
```

Comment:
If this installation method is meant for folks to simply install `dbt` as opposed to contributing and developing on `dbt`, then I'd recommend using this method instead of `editable_requirements.txt`.